src/module/actor/actor-sheet.js

```diff
@@ -534,7 +534,19 @@ export class OseActorSheet extends ActorSheet {
     });
 
     html.find(".item-delete").click((event) => {
-      this._removeItemFromActor(event);
+      const item = this._getItemFromActor(event);
+      const itemData = item?.system;
+
+      if (item.type === "container" && itemData.itemIds.length > 0) {
```

Comment:
This is more personal preference than anything, but I'm generally a fan of as few branches as possible:

```js
if (item.type !== 'container' || !item?.system?.itemIds?.length)
  return this._removeItemFromActor(event);
  
Dialog.confirm({...});  
```

Reply:
I agree - fixed in latest commit!